Makefile

```
@@ -22,7 +22,10 @@ vet: ## Run go vet against code.
 	go vet ./cmd/... ./pkg/...
 
 test: fmt vet envtest ## Run tests.
-	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./pkg/...
+	go test -v ./pkg/...
+
+e2etest: fmt vet envtest ## Run tests.
+	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./pkg/... ./test/...
```

this was technically running both the e2e's and unit tests.
```suggestion
	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) -p path)" go test -v ./test/...
```